tensorflow/contrib/image/python/ops/dense_image_warp.py

```diff
@@ -60,28 +60,34 @@ def _interpolate_bilinear(grid,
       msg = 'Grid must be 4 dimensional. Received size: '
       raise ValueError(msg + str(grid.get_shape()))
 
-    batch_size, height, width, channels = shape
+    batch_size, height, width, channels = \
```

Comment:
I'm not sure what the official tensorflow style is here, but I find parentheses instead of backslashes more readable to make the tuple wrap. Something like,

```suggestion
    batch_size, height, width, channels = (array_ops.shape(grid)[0],
                                           array_ops.shape(grid)[1],
                                           array_ops.shape(grid)[2],
                                           array_ops.shape(grid)[3])
```

Reply:
After I read the google python style guideline, I believe parentheses is more preferable. Thanks for the suggestion.